src/io.jl

```diff
@@ -223,10 +221,13 @@ function prepare_gen_data(
     keys = ["coords", "volt", "tech", "cap", "pfactor", "minload", "scap",
```

Comment:
`keys` is a function, so best to avoid that name
```suggestion
     _keys = ["coords", "volt", "tech", "cap", "pfactor", "minload", "scap",
```